src/main/webapp/script.js

```diff
@@ -211,15 +213,17 @@ function convertRangeToString(range) {
  * @param {string} idPrefix prefix of div id to be altered
  */
 function showLoadingGif(idPrefix) {
-  document.getElementById(idPrefix + 'loading-img').style.display = 'block';
+  document.getElementById(idPrefix + 'video-results-loading-container')
+      .style.visibility = 'visible';
 }
 
 /**
  * Hide loading image
  * @param {string} idPrefix prefix of div id to be altered
  */
 function hideLoadingGif(idPrefix) {
-  document.getElementById(idPrefix + 'loading-img').style.display = 'none';
+  document.getElementById(idPrefix + 'video-results-loading-container')
+      .style.visibility = 'hidden';
 }
 
```

Comment:
Update the display to visibility to ensure the whole div gets hidden/displayed; 
Tested and worked better than simply display = 'none'